src/sidebar/services/load-annotations.js

```diff
@@ -16,53 +16,73 @@ export default function loadAnnotationsService(
   let searchClient = null;
 
   /**
-   * Load annotations for all URIs and groupId.
+   * Load all annotations that match `options` criteria
    *
-   * @param {string[]} uris
-   * @param {string} groupId
+   * @param {Object} options
+   *   @param {string} options.groupId
+   *   @param {string[]} [options.uris]
    */
-  function load(uris, groupId) {
+  function load(options) {
```

Comment:
Alternatively:

```suggestion
  function load({groupId, uris = undefined}) {
```

Reply:
Or:

```suggestion
  function load({groupId, uris = []}) {
```

Reply:
Alas, this change doesn't do well with the current structure of the code, as I do need a reference to the entire `options` object in the `load` function (as well as selective destructured references).